tests/integration/store/test_component_props.py

```diff
@@ -0,0 +1,69 @@
+import json
+import time
+from pytest import approx
+import dash
+from dash.dependencies import Input, Output, State
+from dash.exceptions import PreventUpdate
+import dash_core_components as dcc
+import dash_html_components as html
+
+
+def test_stcp100_clear_data_on_all_types(store_app, dash_duo):
+    dash_duo.start_server(store_app)
+
+    assert dash_duo.wait_for_contains_text("#output", store_app.uuid)
+
+    dash_duo.multiple_click("#btn", 3)
+    assert dash_duo.get_local_storage() == {"n_clicks": 3}
+
+    # button click sets clear_data=True on all type of stores
+    dash_duo.find_element("#clear-btn").click()
+
+    assert (
+        not dash_duo.find_element("#output").text
+        and not dash_duo.get_local_storage()
+        and not dash_duo.get_session_storage()
+    ), "set clear_data=True should clear all data in three storage types"
+
+
+def test_stcp200_modified_ts(store_app, dash_duo):
```

Comment:
stcp200, stdl001 were part of one gigantic test, I split them into smaller tests with specific focus